src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/noc/documents/NocDocumentServiceBaseSetup.java

```diff
@@ -28,7 +28,7 @@
 import static uk.gov.hmcts.reform.finrem.caseorchestration.service.noc.documents.NocDocumentService.CASE_DATA;
 import static uk.gov.hmcts.reform.finrem.caseorchestration.service.noc.documents.NocDocumentService.CASE_DETAILS;
 
-public class NocDocumentServiceBaseTest {
+public class NocDocumentServiceBaseSetup {
```

Comment:
Same for this, if it is a test class or to be extended by a test class, you probably want to call it a test class, otherwise a quick glance might trick you this is a class from the main project